datumaro/plugins/data_formats/video.py

```diff
@@ -138,7 +138,23 @@ class VideoKeyframesImporter(VideoFramesImporter):
     def build_cmdline_parser(cls, **kwargs):
         parser = super().build_cmdline_parser(**kwargs)
         parser.add_argument(
-            "-t", "--threshold", type=float, default=0.3, help="Frame step (default: %(default)s)"
+            "-t",
+            "--threshold",
+            type=float,
+            default=0.3,
+            help="Similarity threshold (default: %(default)s)",
+        )
+        return parser
+
+    @classmethod
+    def build_cmdline_parser(cls, **kwargs):
+        parser = super().build_cmdline_parser(**kwargs)
```

Comment:
```suggestion
```

It seems that those lines should be removed not to declare `build_cmdline_parser()` again.

Reply:
thank you for pointing out this!